zappa/cli.py

```diff
@@ -2080,6 +2095,11 @@ def load_settings(self, settings_file=None, session=None):
         self.xray_tracing = self.stage_config.get('xray_tracing', False)
         self.desired_role_arn = self.stage_config.get('role_arn')
 
+        # Load ALB-related settings
+        self.use_alb = self.stage_config.get('alb_enabled', False)
+        self.alb_vpc_config = self.stage_config.get('alb_vpc_config', {})
+        self.alb_lambda_alias = 'current-alb-version' if self.use_alb else None
```

Comment:
TODO: [Change this to a constant.](https://github.com/youcandanch/Zappa/commit/9d13780daa1f5cf22f48ebc4b9e25ae5f79ba83e#r32557898)